lib/generators/dockerfile_generator.rb

```diff
@@ -550,6 +554,22 @@ def using_solidq?
     @gemfile.include?("solid_queue") and includes_jobs?
   end
 
+  def solidq_launcher
+    if !using_solidq?
+      :none
+    elsif deploy_database != "sqlite3"
+      :process
+    elsif File.exist? "config/puma.rb"
+      if File.read("config/puma.rb").include?("SOLID_QUEUE_IN_PUMA")
+        File.exist?("fly.toml") ? :env : :none
+      else
+        :puma
+      end
+    else
+      :procfile
+    end
+  end
```

Comment:
Any plans to support Postgres specifically in the future?

Reply:
Are there issues with Postgres?  Ideally Solid Queue (like Sidekiq) runs as a separate machine; something that is generally not possible with Sqlite3.

Reply:
Gotcha. I was thinking about having it be on the same machine, but that's probably not a good idea. Thanks.